news/4172.feature

```diff
@@ -0,0 +1 @@
+Introduces custom permission to manage recyclebin @rohnsha0
```

Comment:
```suggestion
Introduces custom permission to manage recycle bin. @rohnsha0
```

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4176/commits/82b0107e5d315932ef78dec8d0f88bb953fa76d3